app/assets/javascripts/discourse/app/components/create-invite-uploader.js

```diff
@@ -0,0 +1,112 @@
+import Component from "@ember/component";
+import getUrl from "discourse-common/lib/get-url";
+import discourseComputed from "discourse-common/utils/decorators";
+import {
+  displayErrorForUpload,
+  validateUploadedFiles,
+} from "discourse/lib/uploads";
+
+export default Component.extend({
+  tagName: "span",
```

Comment:
If possible we try to go as tag less as possible now, I think @CvX can confirm?

Reply:
I think you are right.

Reply:
Yup, that's The Glimmer Way. Going tagless now saves us work later 😃